### ISSUE TITLE
Placed SF blocks revert to previously placed blocks.

### ISSUE THREAD

**Issue by Boomer-1** [Slimefun/Slimefun4#4108]
### ❗ Checklist

- [X] I am using the official english version of Slimefun and did not modify the jar.
- [X] I am using an up to date "DEV" (not "RC") version of Slimefun.
- [X] I am aware that issues related to Slimefun addons need to be reported on their bug trackers and not here.
- [X] I searched for similar open issues and could not find an existing bug report on this.

### 📍 Description

After placing any slimefun block, at some point in the future if you break it and later on place another block in the same place, the first sf machine that was placed will drop from that spot and the 2nd machine you placed will break and become useless.

### 📑 Reproduction Steps

there is no consistent method to reproduce this problem yet, but a lot of the devs in discussions came up with 2 possible causes. The first was block storage.
The second that came up was that because worlds have thousands more machines than they used to years ago, that there's too many slimefun items to get all the data saved before the server shuts down.

It's been discussed that it seems to happen a lot more often with items placed within the 10 minutes before a server restart. Unfortunately no one has been able to consistently confirm this. 

Networks and IE have been the subject of this discussion quite frequently, but its because they are 2 of the most commonly used addons, however it's been brought up in the discord with core and multiple addons having this happen.

### 💡 Expected Behavior

no items should leave sf data behind after breaking  to cause this. when sf machines are mined, it should be as if nothing was ever there.

### 📂 `/error-reports/` folder

no error reports get generated.

### 💻 Server Software

Paper

### 🎮 Minecraft Version

1.20.x

### ⭐ Slimefun version

been happening for some time.

**Comment by YukiMuori** [comment]
Same problem here.

**Comment by iBreeze7** [comment]
I encountered this problem a long time ago, and now I am encountering it again

**Comment by SrMonsterYT** [comment]
Hi, I'm experiencing the same issue, but in my case, it's even worse: the machines actually get duplicated.

Here’s what happens:

I place a Slimefun machine (from any addon).
I break it, and the machine disappears as expected.
Then I place any regular block in the same spot.
After restarting the server, I break that regular block, and the machine drops again, even though I already broke and removed it earlier.

So basically, players can duplicate machines just by doing this before a server restart.
Is there any known fix or workaround for this?
